### ISSUE TITLE
Diagramme im ZEIT Energiemonitor

### ISSUE THREAD

**Issue by gwater** [stefanw/bibbot#319]
Anscheinend verträgt sich bibbot nicht mit den Diagrammen im ZEIT Energiemonitor. Wenn Bibbot aktiv ist, werden die Diagramme in Firefox nicht dargestellt. Deaktiviert man bibbot, gehen die Diagramme wieder.

**Comment by awendt** [comment]
Link: https://www.zeit.de/wirtschaft/energiemonitor-deutschland-gaspreis-spritpreis-energieversorgung

**Comment by Reselatte** [comment]
Oh, wie kann man BibBot deaktivieren?

**Comment by awendt** [comment]
@Reselatte Firefox: Application Menu (3 Striche) > Addons and themes > BibBot deaktivieren

![image](https://github.com/stefanw/bibbot/assets/11964/18db9e3f-4bbb-44af-8712-05617c333b9b)

**Comment by stefanw** [comment]
Einzelne Seiten lassen sich auch deaktivieren: in den BibBot-Einstellungen unter „Weitere Einstellungen“. Weiß aber nicht, ob das hier die Probleme mit den Diagrammen behebt.

**Comment by awendt** [comment]
Das nutzt hier leider nichts, weil der Bot für zeit.de sehr hilfreich ist. Ich möchte ihn daher nicht deaktivieren.

Mein Workaround ist daher ein Inkognito-Fenster in Kombination mit "Run in Private Window = Don't Allow".

**Comment by gwater** [comment]
Ja, es bräuchte einen Filter für komplette URLs, nicht nur ganze Domains

**Comment by stefanw** [comment]
Naja, idealerweise kommen sich extension und Seiten-scripte nicht in die Quere. Aber warum das hier so ist, konnte ich bisher nicht herausfinden.
